straxen/contexts.py

```diff
@@ -45,8 +45,9 @@
         mv_blank=(1999, 1999),
         nveto=(2000, 2119),
         nveto_blank=(2999, 2999)),
-    nn_architecture='mlp_model.json',
-    nn_weights='mlp_model.h5',
+    nn_architecture=None,
```

Comment:
Better overwrite default to None I think such to make clear that this option is not used

Reply:
Looks good to me. It might be ok to just keep these two options too, as long as there is a `nn_file` these are not used.